app/assets/javascripts/paging_schedule_updater.js

```diff
@@ -0,0 +1,80 @@
+var pagingScheduleUpdater = (function() {
+
+  var defaultOptions = {
+    selector: '[data-paging-schedule-updater="true"]',
+    urlSelector: '[data-scheduler-lookup-url]',
+    textSelector: '[data-scheduler-text]',
+    singleLibrarySelector: '[data-single-library-value]'
+  };
+
+  return {
+    init: function(opts) {
+      var _this = this;
+      _this.options = $.extend(defaultOptions, opts);
+      $(document).on('ready page:load', function(){
+        _this.requestPagingScheduleData(); // Set initial schedule state
+        _this.addSelectChangeBehavior();
+      });
+    },
+
+    options: {},
+
+    addSelectChangeBehavior: function() {
+      var _this = this;
+      _this.selectElement().on('change', function() {
+        _this.requestPagingScheduleData();
+      });
+    },
+
+    requestPagingScheduleData: function() {
+      var _this = this;
+      if ( _this.schedulerValue() && _this.schedulerText().length > 0 ) {
+        $.ajax({
```

Comment:
Do we need some kind of spinner when we're loading the data (in case the request takes some time..)?